parameter/TypeElement.cpp

```diff
@@ -76,8 +76,9 @@ bool CTypeElement::hasMappingData() const
 // Element properties
 void CTypeElement::showProperties(std::string& strResult) const
 {
-    (void)strResult;
-    // Prevent base from being called in that context!
+    showDescriptionProperty(strResult);
+    // Prevent base from being called from the Parameter Type context.
```

Comment:
what does the parameter type has to do here ? TypeElement does not know Parameter Type. Why do we need to override showProperties ? Element already print the description.